src/drt/src/TritonRoute.tcl

```diff
@@ -460,6 +460,10 @@ proc detailed_route_set_unidirectional_layer { args } {
   drt::detailed_route_set_unidirectional_layer $args
 }
 
+proc fix_max_spacing { args } {
+  sta::check_argc_eq0 "detailed_route_set_unidirectional_layer" $args
```

Comment:
wrong command name in check_argc_eq0

Reply:
Actually this command was added for debugging purposes only. I don't see an effective usage for it. Should I keep it?